dropwizard-lifecycle/src/main/java/io/dropwizard/lifecycle/PortDescriptor.java

```diff
@@ -0,0 +1,69 @@
+package io.dropwizard.lifecycle;
+
+import com.google.common.base.MoreObjects;
+
+public class PortDescriptor {
```

Comment:
could the class and all of the properties be made `final` so the class is effectively immutable?

Reply:
I can certainly do that. I will make that change on the resubmitted pull request against master.

Reply:
The pull on master is up with the discussed changes. Thanks!